### PR TITLE
chore: add collateralBrand to vault and auction trace() calls

### LINKED ISSUE #7718: Ensure all vaultManager trace/log messages include collateral brand
## What is the Problem Being Solved?

While debugging, we saw many logs of [`considering liquidation`](https://github.com/Agoric/agoric-sdk/blob/2d07ae91fb96f526b5af9a3efb52e64df996221f/packages/inter-protocol/src/vaultFactory/vaultManager.js#L1181) and it would have been nice to know whether they were from one vault or many. 

## Description of the Design

Ensure vaultManager logs and `trace()` calls include the collateral brand.

## Security Considerations

None

## Scaling Considerations

None

## Test Plan

None

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
closes: #7718

## Description

Add collateralBrand to vault and auction trace() calls

### Security Considerations

make it easier to track down where something is going wrong (or right).

### Scaling Considerations

Helpful as we add more collateral types.

### Documentation Considerations

None

### Testing Considerations

None

### CODE REVIEW

File: packages/inter-protocol/src/auction/auctionBook.js [hunk 1]
```diff
@@ -358,7 +358,7 @@ export const prepareAuctionBook = (baggage, zcf, makeRecorderKit) => {
         ) {
           const { priceBook, curAuctionPrice } = this.state;
           const { helper } = this.facets;
-          trace('acceptPrice');
+          trace('acceptPrice', this.state.collateralBrand);
```
Comment (turadg) [inline packages/inter-protocol/src/auction/auctionBook.js]: consider putting it at the beginning for hierarchy in the logs (Vault manager: brand: unique message)

File: packages/inter-protocol/src/vaultFactory/vault.js [hunk 2]
```diff
@@ -350,7 +350,8 @@ export const prepareVault = (baggage, makeRecorderKit, zcf) => {
           }
           const { phase } = state;
           const uiState = facets.helper.getStateSnapshot(phase);
-          trace('updateUiState', state.idInManager, uiState);
+          const brand = facets.helper.collateralBrand();
+          trace('updateUiState', state.idInManager, uiState, brand);
```
Comment (turadg) [inline packages/inter-protocol/src/vaultFactory/vault.js]: oh yeah, the unique id before wasn't really unique (over the factory)

are there any other vault-level traces that need these?

File: packages/inter-protocol/src/auction/auctionBook.js [hunk 3]
```diff
@@ -404,7 +404,7 @@ export const prepareAuctionBook = (baggage, zcf, makeRecorderKit) => {
           maxBuy,
           { trySettle, exitAfterBuy = false },
         ) {
-          trace('accept scaledBid offer');
+          trace('accept scaledBid offer', this.state.collateralBrand);
```
Comment (turadg) [inline packages/inter-protocol/src/auction/auctionBook.js]: maybe worth having a `trace` or `log` method on the helper facet

**Review by turadg — APPROVED** [review]

File: packages/inter-protocol/src/vaultFactory/vault.js (hunk 2 above)
Comment (Chris-Hibbert) [reply packages/inter-protocol/src/vaultFactory/vault.js]: I looked through vaults and didn't add details when there were consecutive `traces` without an intervening `await`.

File: packages/inter-protocol/src/auction/auctionBook.js (hunk 1 above)
Comment (Chris-Hibbert) [reply packages/inter-protocol/src/auction/auctionBook.js]: a fine idea.

File: packages/inter-protocol/src/auction/auctionBook.js (hunk 3 above)
Comment (Chris-Hibbert) [reply packages/inter-protocol/src/auction/auctionBook.js]: didn't seem worth it to me.

**Pull request merged**